### ISSUE TITLE
[BUG] Clamav-freshclam malfunction not getting the update

### ISSUE THREAD

**Issue by stdevPavelmc** [stdevPavelmc/mailad#68]
Some bugs was found in the latest commits about clamav & freshclam

- `DatabaseMirror` parameter can be expressed multiple times, if https you can omit the prefix, but no for http
- We must increase the timeouts as in slow networks this can be a problem and the update never finish

**Comment by stdevPavelmc** [comment]
Check comments about the config parameters in https://linux.die.net/man/5/freshclam.conf

**Comment by stdevPavelmc** [comment]
A sample of a working configuration from a restrictive location (less proxy config for client privacy)

```
# Automatically created by the clamav-freshclam postinst
# Comments will get lost when you reconfigure the clamav-freshclam package

DatabaseOwner clamav
UpdateLogFile /var/log/clamav/freshclam.log
LogVerbose false
LogSyslog false
LogFacility LOG_LOCAL6
LogFileMaxSize 0
LogRotate true
LogTime true
Foreground false
Debug false
MaxAttempts 5
DatabaseDirectory /var/lib/clamav
DNSDatabaseInfo current.cvd.clamav.net
ConnectTimeout 300
ReceiveTimeout 3600
TestDatabases yes
ScriptedUpdates yes
CompressLocalDatabase no
SafeBrowsing false
Bytecode true
NotifyClamd /etc/clamav/clamd.conf
# Check for new database 24 times a day
Checks 24
DatabaseMirror http://103.230.76.202/clamav
DatabaseMirror http://mirror.webtastix.net/clamav
DatabaseMirror http://repo.virtualhosting.hk/clamavdb
DatabaseMirror https://mirrors.mediatemple.net/freshclam
```

**Comment by stdevPavelmc** [comment]
closed via #69
